events/HF_religious_events.txt

```diff
@@ -10486,41 +10486,72 @@ character_event = {
 	is_triggered_only = yes
 
 	immediate = { 
-		killer = { 
-			#Bonus points first.
-			if = { #The gods appreciate your sacrifice.
-				limit = {
-					is_adult = no
-					ROOT = { is_child_of = PREV } 
+		killer = {
+
+			# Warcraft
+			# Racial points
+			if = { # Has more blood than small humanoids.
+				limit = { ROOT = { is_medium_humanoid_trigger = yes } }
+				change_variable = { which = burned_hearts_points value = 1 } # 2 total for medium humanoid
+			}
+			else_if = { # Has more blood than medium humanoids.
+				limit = { ROOT = { is_large_humanoid_trigger = yes } }
+				change_variable = { which = burned_hearts_points value = 2 } # 3 total for large humanoid
+			}
+			else_if = { # Dragons are magical!
+				limit = { ROOT = { is_dragon_race_trigger = yes } }
+				if = {
+					limit = { ROOT = { is_whelp_trigger = yes } }
+					change_variable = { which = burned_hearts_points value = 2 } # 3+1 bonus total for whelp
+				}
+				else_if = {
+					limit = { ROOT = { is_drake_trigger = yes } }
+					change_variable = { which = burned_hearts_points value = 5 } # 6 total for drake
 				}
+				else_if = {
+					limit = { ROOT = { is_dragon_trigger = yes } }
+					change_variable = { which = burned_hearts_points value = 7 } # 8 total for dragon
+				}
+				else = { change_variable = { which = burned_hearts_points value = 9 } } # 10 total for wyrm
+				if = {
+					limit = { ROOT = { is_aspect_trigget = yes } }
+					change_variable = { which = burned_hearts_points value = 20 } # +20 if it is aspect
+				}
+			}
+			# Bonus points
+			if = { # Innocent souls!
+				limit = { ROOT = { is_adult = no } }
 				change_variable = { which = burned_hearts_points value = 1 }
 			}
-			if = { #And a divine child is even better.
-				limit = {
-					is_adult = no
-					ROOT = { religion = PREV } 
-					OR = { 
-						has_religion_feature = religion_holy_family 
-						has_religion_feature = religion_feature_zun
-					} 
-				}
+			if = { #The gods appreciate your sacrifice.
+				limit = { ROOT = { is_child_of = PREV } }
 				change_variable = { which = burned_hearts_points value = 1 }
+
+				if = { #And a divine child is even better.
+					limit = {
+						OR = {
+							has_religion_feature = religion_holy_family
+							has_religion_feature = religion_feature_zun
+						} 
+					}
+					change_variable = { which = burned_hearts_points value = 1 }
+				}
```

Comment:
Hmm. In vanilla, it "gives" more points even if the divine child isn't yours. This change is intended?

Reply:
Yes, intended.